log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/moment.js

```diff
@@ -0,0 +1,2808 @@
+//! moment.js
+//! version : 2.8.1
```

Comment:
Also I see the same library is used in many gadgets, and in that case this can be included in the portal/lib and then referred from many gadgets.